### DISCUSSION TITLE
Issue with your-first-extension starter

### DISCUSSION THREAD

**Discussion by Akshay090** [microsoft/vscode-discussions#93] [Extension Development QnA] [answered]
Hi Folks, I've been trying to get the starter extension working from this [guide](https://code.visualstudio.com/api/get-started/your-first-extension) on my system, but stuck with this issue
system : mac os version - 12.5
node version - 12 

**Issue**
Extension doesn't load in Extension development host window

**Debugging** till now

<img width="1259" alt="Screenshot 2022-08-06 at 7 56 35 PM" src="https://user-images.githubusercontent.com/28626499/183253282-a8e5c67d-0862-4ea6-8bb5-fcd09b2c574e.png">

from what i checked till now it looks something related to telementary api
```
Extension 'vscode.git' CANNOT use API proposal: telemetry.
```
this error occurrs in ``` /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:89```

While looking into similar issues - https://github.com/microsoft/vscode/issues/97181 resolution was to use insider build, but nothing like that is mentioned in the starter [guide](https://code.visualstudio.com/api/get-started/your-first-extension), so mostly this is a different issue

**Comment by Akshay090** [comment, marked as answer]
Installed VS Code Insider and tried, the starter extension is loading correctly, 

if someone can confirm if vscode insider is needed to extension dev it'll be really great 🙌

**Reply by alefragnani** [reply]
Yep. Any **Proposed API** requires **Insiders** release to play with.

More info about Proposed API can be found here https://code.visualstudio.com/api/advanced-topics/using-proposed-api

**Reply by alefragnani** [reply]
But, just to be clear, to create _regular_ extensions you don't need Insiders release. You can use the Stable (as I do most of the time). 

I'm not sure what which additional sample are you following, but the message you provide refers to `vscode.git` and its use of `API proposal: telemetry`. This _proposal_ is the reason that forces you to use Insiders release.

Hope this helps

**Reply by Akshay090** [reply]
@alefragnani thanks that helps, i was following the basic JS boiler plate, 
whenever i try to run the extension this error is shown, this looks like something specific to my system, 
the existing extensions like the builtin - vscode git, microsoft authentication etc, are causing this error
for now I'm able to continue my extension dev in vs code insider 😄
